mysqld-mixin/rules/rules.yaml

```diff
@@ -0,0 +1,15 @@
+groups:
+- name: mysqld_rules
+  rules:
+
+  # Record slave lag seconds for pre-computed timeseries that takes
+  # `mysql_slave_status_sql_delay` into account
+  - record: :mysql_slave_lag_seconds
```

Comment:
I'm not a fan of making metric names start with a bare `:`. Is this a mixin pattern? Typically if I am not using an aggregation, I start rules like this with `instance`. So, `instance:mysql_slave_lag_seconds`.

Reply:
I'm not a fan either - the rule I was applying was to make sure all recording rules have a colon in.  I like your suggestion though.